js/symb/taxa.taxonomyloader.js

```diff
@@ -195,3 +195,32 @@ function acceptanceChanged(f){
 	if(accStatusObj[0].checked) document.getElementById("accdiv").style.display = "none";
 	else document.getElementById("accdiv").style.display = "block";
 }
+
+// listener for taxon rank
+
+document.getElementById('rankid').addEventListener('change', function () {
+	var selectedValue = this.value; // Get the chosen value
```

Comment:
Not important to change but worth mentioning that `var` has been largely replaced by `const` and `let` in modern js. `var` has global scoping, while the other two have clearer names that explicit about their usage. Just FYI.